### DISCUSSION TITLE
Welcome to ATA-Site Discussions!

### DISCUSSION THREAD

**Discussion by Malvern97** [Malvern97/ATA-Site#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Malvern97** [comment]
The web also requires url security

**Reply by khumalothembi** [reply]
True security is part and parcel, it can be added to the components and we need a file that the team works on
